modules/gdscript/gdscript_editor.cpp

```diff
@@ -2148,7 +2148,7 @@ static void _find_identifiers(const GDScriptCompletionContext &p_context, bool p
 	}
 
 	const GDScriptParser::ClassNode *clss = p_context._class;
-	bool _static = !p_context.function || p_context.function->_static;
+	bool _static = !p_context.function ? false : p_context.function->_static;
```

Comment:
This is equivalent to `p_context.function && p_context.function->_static`, no?

Reply:
The problem arises when p_context.function is a nullptr. In that case, p_context.function->_static will give erroneous/incorrect values. That's why, in the expression, it first checks if p_context.function is a nullptr

Reply:
`&&` short-circuits: if `p_context.function` evaluates to false, `p_context.function->_static` will not be evaluated.

Reply:
Oh yeah! How could I miss it, the short-circuiting thing. Thanks, I'll update it.